Makefile

```diff
@@ -31,8 +32,10 @@ lint:
 unit:
 	cargo test --bins --no-fail-fast
 
-integration: release # It needs to be release so we correctly test the RSS usage
-	CONMON_BINARY="$(MAKEFILE_PATH)target/release/$(BINARY)" RUNTIME_BINARY="$(RUNTIME_PATH)" go test -v pkg/client/*
+integration: release-static # It needs to be release so we correctly test the RSS usage
+	export CONMON_BINARY="$(MAKEFILE_PATH)target/x86_64-unknown-linux-musl/release/$(BINARY)" && \
+	export RUNTIME_BINARY="$(RUNTIME_PATH)" && \
+	go test pkg/client/* -v -ginkgo.v
```

Comment:
Running the integration tests with a statically compiled binary results in a much lower accounted RSS:
```
2022-04-04T11:37:13.3038025Z VmRSS for server is 172
2022-04-04T11:37:13.4873763Z VmRSS for server is 172
2022-04-04T11:37:13.6331590Z VmRSS for server is 172
2022-04-04T11:37:13.8101498Z VmRSS for server is 172
2022-04-04T11:37:14.0258265Z VmRSS for server is 172
2022-04-04T11:37:14.2525484Z VmRSS for server is 176
2022-04-04T11:37:14.4764609Z VmRSS for server is 172
2022-04-04T11:37:17.7302650Z VmRSS for server is 176
2022-04-04T11:37:17.9593723Z VmRSS for server is 180
2022-04-04T11:37:18.1867412Z VmRSS for server is 176
2022-04-04T11:37:18.4142868Z VmRSS for server is 176
2022-04-04T11:37:21.6480393Z VmRSS for server is 176
```

Reply:
Awesome :)